rest-api/flows-trigger-file.sh

```
@@ -1,37 +1,58 @@
+  echo "1) File upload:"
+  echo "   $0 'document-processing-workflow-id' file './contract.pdf' documentType=contract priority=high"
+  echo ""
+  echo "2) URL processing:"
+  echo "   $0 'image-analysis-workflow-id' url 'https://storage.example.com/products/laptop-x1.jpg' imageType=product category=electronics"
+  echo ""
+  echo "3) Complex parameters:"
+  echo "   $0 'document-processing-workflow-id' file './contract.pdf' customerId=CID-12345 department=legal requireSignature=true processingMode=detailed"
+  exit 1
+fi
 source_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 if [ -f "$source_dir/.env" ]; then export $(grep -v '^#' "$source_dir/.env" | xargs); fi
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)